app/components/Footer.tsx

```diff
@@ -1,10 +1,165 @@
-    <p>
-      built by{' '}
-      <a href="https://swapr.eth.limo" className="text-text-primary-main">
-        Swapr.eth
+  <div className="mt-20 border-t border-surface-surface-2 bg-surface-surface-0 px-6 md:px-10 lg:px-20 xl:px-40">
+    <div className="flex flex-col-reverse items-start justify-between py-10 md:flex-row">
+      <div className="w-full space-y-12 md:w-fit">
+        <div className="hidden space-y-2 md:block">
+          <Logo />
+          <p>Where prediction markets and AI agents meet</p>
+        </div>
+        <PoweredByPartners />
+      </div>
+      <div className="flex space-x-10">
+        <div>
+          <p className="mb-2 text-lg font-bold">Markets</p>
+          <ul className="space-y-1">
+            {Object.values(Categories).map(category => (
+              <li className="text-md font-medium capitalize" key={category}>
+                <a href={`/?c=${category}`}>
+                  <p className="hover:text-text-primary-main">{category}</p>
+                </a>
+              </li>
+            ))}
+          </ul>
+        </div>
+        <div>
+          <p className="mb-2 text-lg font-bold">Socials</p>
+          <ul className="space-y-1">
+            <li className="text-md capitalize">
+              <a href={X_URL} target="_blank">
+                <p className="hover:text-text-primary-main">X</p>
+              </a>
+            </li>
+            <li className="text-md capitalize">
+              <a href={DISCORD_URL} target="_blank">
+                <p className="hover:text-text-primary-main">Discord</p>
+              </a>
+            </li>
+          </ul>
+        </div>
+      </div>
+    </div>
+    <MadeBySwapr />
+  </div>
+);
+
+const PoweredByPartners = () => (
+  <div className="flex flex-col space-y-2 rounded-8 bg-surface-surface-1 p-3 md:flex-row md:items-center md:space-x-2 md:space-y-0">
+    <p className="mb-2 text-nowrap md:mb-0 md:mr-2">Powered by</p>
+    <div className="grid grid-cols-2 gap-2 md:flex">
+      <a
+        href="https://www.gnosis.io/"
+        className="flex items-center space-x-1"
+        target="_blank"
+      >
+        <GnosisIcon />
+        <p className="font-bold">Gnosis</p>
+      </a>
+      <span className="hidden md:inline-block">•</span>
+      <a href="https://reality.eth.limo/" className="font-bold" target="_blank">
+        Reality.eth
+      </a>
+      <span className="hidden md:inline-block">•</span>
+      <a
+        href="https://kleros.io/"
+        className="flex items-center space-x-1"
+        target="_blank"
+      >
+        <KlerosIcon />
+        <p className="font-bold">Kleros</p>
+      </a>
+      <span className="hidden md:inline-block">•</span>
+      <a href="https://www.gnosis.io/labs" className="font-bold" target="_blank">
+        Gnosis AI
+      </a>
+    </div>
+  </div>
+);
+
+const MadeBySwapr = () => (
+  <div className="mt-12 flex items-center justify-center border-t border-surface-surface-2 py-5 md:justify-start">
```

Comment:
[MINOR][NON-BLOCKING][OPINIONATED]:
I feel there's a lot of blank space between the footer content and this component, I think the padding is enough here. Some visual examples:

```suggestion
  <div className="flex items-center justify-center border-t border-surface-surface-2 py-5 md:justify-start">
```

### Now (desktop):
![Screenshot 2024-10-22 at 11 24 38](https://github.com/user-attachments/assets/a97731bb-d5de-4826-a8f1-165cc9667353)
### Without margin (desktop):
![Screenshot 2024-10-22 at 11 24 48](https://github.com/user-attachments/assets/04db8df9-7dd1-4550-9df3-7d9569ae3f2c)
-----------

### Now (mobile):
![Screenshot 2024-10-22 at 11 26 10](https://github.com/user-attachments/assets/b5f1f3d3-1c3b-4a49-ac09-e0309be8b4f2)
### Without margin (mobile):
![Screenshot 2024-10-22 at 11 26 17](https://github.com/user-attachments/assets/1625c11f-2c52-452f-a97d-56933dd090b5)

Reply:
On spacing thing, agree it's a bit too much, but I'll keep some space to breath, I'll settle in half: `mt-6`.